package.xml

```diff
@@ -44,30 +44,30 @@
   <run_depend>moveit_ros_planning_interface</run_depend>
   <run_depend>moveit_ros_perception</run_depend>
   <run_depend>interactive_markers</run_depend>
-  <run_depend>moveit_visual_tools</run_depend>
-  <run_depend>rviz_visual_tools</run_depend>
+  <!-- <run_depend>moveit_visual_tools</run_depend> -->
+  <!-- <run_depend>rviz_visual_tools</run_depend> -->
   <run_depend>joint_state_publisher</run_depend>
   <run_depend>robot_state_publisher</run_depend>
   <run_depend>joy</run_depend>
-  <run_depend>pcl_ros</run_depend>
-  <run_depend>pcl_conversions</run_depend>
-  <run_depend>rosbag</run_depend>
+  <!-- <run_depend>pcl_ros</run_depend> -->
+  <!-- <run_depend>pcl_conversions</run_depend> -->
+  <!-- <run_depend>rosbag</run_depend> -->
   <run_depend>rviz</run_depend>
   <run_depend>tf2_ros</run_depend>
   <run_depend>tf2_eigen</run_depend>
   <run_depend>tf2_geometry_msgs</run_depend>
   <run_depend>xacro</run_depend>
 
   <test_depend>moveit_resources_panda_moveit_config</test_depend>
```

Comment:
I'm actually a bit confused that we have a test dependency to `moveit_resources_panda_moveit_config` while the run dependency uses `panda_moveit_config`...

Reply:
I was confused by this too, but didn't want to remove it. Right now no tests are enabled so we could remove this and expect the person that ports the [test tutorial](https://github.com/ros-planning/moveit2_tutorials/tree/main/doc/tests) to add it back if needed?? Any suggestions on how you would like to move forward here?